src/LeftSidebar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import VerticalSlider from "./VerticalSlider";
+
+function openLeftSideBar() {
+    document.getElementById("leftSideBar").style.width = "250px";
```

Comment:
Instead of using `document.getElementById` here, you can use React to handle the state and conditionally apply the class controlling the width using the `value ? thisIfTrue : thisIfFalse` syntax.

That would look something like this:
```
const LeftSidebar = ({ setSize, setOpacity }) => {
    const [isOpen, setIsOpen] = React.useState(true);

  return (
    <div className=''>
      <aside
        id='leftSideBar'
        name='leftSideBar'
        className={`min-h-full bg-gray-400 p-4 fixed left-0 rounded-md items-center border-2 ${
          isOpen ? 'w-40' : 'w-0'
        }`}
      >
...
 ```
 
 and you could change that state within a button, for example:
 
 ```
 <button className='top-0 w-full' onClick={() => setIsOpen(!isOpen)}>
    toggleSidebar
</button>
 ```

Reply:
You can also hide the sliders when it's closed using the `value && thisIfTrue` syntax.

```
          {isOpen && (
            <>
              <VerticalSlider
                onChange={handleSizeChange}
                sliderName='BrushSize'
                sliderLabel='Brush Size'
                min={1}
                max={100}
              />
              <VerticalSlider
                onChange={handleOpacityChange}
                sliderName='BrushOpacity'
                sliderLabel='Brush Opacity'
                min={0}
                max={100}
              />
            </>
          )}
          ```